package.json

```diff
@@ -46,32 +46,32 @@
   },
   "homepage": "https://github.com/ethereum/solc-js#readme",
   "dependencies": {
-    "command-exists": "^1.2.8",
-    "commander": "^8.1.0",
-    "follow-redirects": "^1.12.1",
-    "js-sha3": "0.8.0",
-    "memorystream": "^0.3.1",
-    "semver": "^5.5.0",
-    "tmp": "0.0.33"
+    "command-exists": ">=1.1.0",
+    "commander": ">=8.0.0 <12.0.0",
```

Comment:
Versions `>=13.0.0` does not pass tests. We could probably support it if we wanted but it would require adjustments in the code.

Versions `>=12.0.0` trigger failures on older node.js versions due to the use of `??` operator or its variants.